arch/cortex-m/src/dcb.rs

```diff
@@ -0,0 +1,58 @@
+//! ARM Debug Control Block
+//!
+//! <https://developer.arm.com/documentation/ddi0403/latest>
```

Comment:
```suggestion
//! <https://developer.arm.com/documentation/ddi0403/latest>
//! Implementation matches `ARM DDI 0403E.e`
```

Not that this doc actually changes in substantial ways that often, but it's useful to have a pointer to which version is actually implemented here.